packages/remix-react/index.tsx

```diff
@@ -58,6 +58,8 @@ export type { ShouldReloadFunction, HtmlMetaDescriptor } from "./routeModules";
 
 export { ScrollRestoration } from "./scroll-restoration";
 
+export type { SerializeFrom } from "./serialize";
```

Comment:
Here's where we make `SerializeFrom` into a public API importable from `@remix-run/react`.